src/app/components/Text/index.jsx

```diff
@@ -1,19 +1,24 @@
 import React from 'react';
+import { string, bool, shape } from 'prop-types';
 import Markdown from 'markdown-to-jsx';
-import {
-  textModelPropTypes,
-  textModelDefaultProps,
-} from '../../models/propTypes/text';
 
-const Text = ({ blocks }) =>
-  blocks.map(({ blockId, model }) => (
-    <Markdown key={blockId} options={{ forceBlock: true }}>
-      {model.text}
-    </Markdown>
-  ));
+const Text = ({ text, options }) => {
+  if (!text) return null;
```

Comment:
In my opinion, this logic should be handled in the container

Reply:
Agreed, won't do that in this PR though.